scout/adapter/mongo/query.py

```diff
@@ -189,50 +189,14 @@ def _get_similar_cases(self, query, institute_ids):
         return select_cases
 
     def build_query(
-        self, case_id, query=None, variant_ids=None, category="snv", build="37"
+        self,
+        case_id: str,
+        query: Optional[dict] = None,
+        variant_ids: Optional[List[str]] = None,
+        category: str = "snv",
+        build: str = "37",
     ) -> dict:
-        """Build a mongo query
-
```

Comment:
This is no longer up to date, so I'd remove it

Reply:
It's just a comment, explaining what these parameters are. But they change all the time and we forget to update the comment. So yeah, I'd just remove it